components/brave_today/browser/direct_feed_controller.cc

```diff
@@ -442,7 +426,6 @@ void DirectFeedController::OnResponse(
   // Parse response data
   auto* loader = iter->get();
   auto response_code = -1;
-  std::string charset = GetResponseCharset(loader);
```

Comment:
Can we rename this enclosing method to "DirectFeedController::OnDownloadFeed" for clarity?